.github/workflows/run_edps.yaml

```
@@ -84,5 +102,4 @@ jobs:
           edps -lw
           edps -w metis.metis_wkf -i $SOF_DATA -c
           edps -w metis.metis_wkf -i $SOF_DATA -lt
-          edps -w metis.metis_wkf -i $SOF_DATA -m all| tee edps.stdout.txt
-          ! grep "'FAILED'" edps.stdout.txt
+          edps -w metis.metis_wkf -i $SOF_DATA -m all | tee edps.stdout.txt ! grep "'FAILED'" edps.stdout.txt
```

edps -w metis.metis_wkf -i $SOF_DATA -m all | tee edps.stdout.txt 
```suggestion
          edps -w metis.metis_wkf -i $SOF_DATA -m all | tee edps.stdout.txt
          ! grep "'FAILED'" edps.stdout.txt
```